runbotics/runbotics-scheduler/src/queue/process-instance/process-instance.scheduler.service.ts

```diff
@@ -47,5 +50,44 @@ export class ProcessInstanceSchedulerService {
             BotWsMessage.TERMINATE,
             processInstanceId
         );
+
+        const queryRunner = this.connection.createQueryRunner();
+        await queryRunner.connect();
+        await queryRunner.startTransaction();
+        try {
```

Comment:
so is query runner mandatory even if we found solution (missing error message from bot)